### DISCUSSION TITLE
MultiColumnListView

### DISCUSSION THREAD

**Discussion by yassinbj3a3d** [dlsc-software-consulting-gmbh/GemsFX#78] [General]
"Hello, I hope you're doing well. I'm a Java developer, and I'm currently working on a project that involves using the MultiColumnListView in JavaFX. However, I'm not very familiar with its functionality, and I haven't been able to implement it in my project successfully. Would you be kind enough to provide me with a brief explanation of how MultiColumnListView works and how I can effectively use it in my project? Your insights would be greatly appreciated. Thank you!

**Comment by dlemmermann** [comment]
You closed the ticket, so I assume you found the demo app?
